spec/views/shared/_header_navbar.html.erb_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+describe 'shared/_header_navbar', type: :view do
+  before do
+    allow(view).to receive(:blacklight_config).and_return(Blacklight.default_configuration)
+    stub_template 'shared/_user_util_links.html.erb' => 'links'
+  end
+
+  it 'has nav links' do
+    render
+    expect(rendered).to have_selector '#user-util-collapse', text: 'links'
```

Comment:
Do we need to have a view test at all?  View tests are deprecated in rails.  Instead can you use a request test, feature test, or a component test?